tests/integration/cmn/client/acl.go

```diff
@@ -0,0 +1,119 @@
+package client
+
+import (
+	"encoding/hex"
+	"fmt"
+	"strconv"
+	"time"
+
+	pb "github.com/anoideaopen/foundation/proto"
+	"github.com/anoideaopen/foundation/test/integration/cmn"
+	"github.com/anoideaopen/foundation/test/integration/cmn/client"
+	"github.com/btcsuite/btcutil/base58"
+	"github.com/golang/protobuf/proto"
+	"github.com/hyperledger/fabric/integration/nwo"
+	"github.com/hyperledger/fabric/integration/nwo/commands"
+	. "github.com/onsi/gomega"
+	"github.com/onsi/gomega/gbytes"
+	"github.com/onsi/gomega/gexec"
+)
+
+const MultisignKeyDelimeter = "/"
+
+func CheckUserMultisigned(network *nwo.Network, peer *nwo.Peer, user *UserFoundationMultisigned) {
+	Eventually(func() string {
+		sess, err := network.PeerUserSession(peer, "User1", commands.ChaincodeQuery{
+			ChannelID: cmn.ChannelAcl,
+			Name:      cmn.ChannelAcl,
+			Ctor:      cmn.CtorFromSlice([]string{"checkKeys", user.PublicKey()}),
```

Comment:
a package tests/common defines constants for ACL functions. "checkKeys" might be common.FnCheckKeys